### DISCUSSION TITLE
Hugo搜索工具hugo-algolia2

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#139] [Announcements]
https://www.bbing.com.cn/202109/hugo-algolia2/

**Comment by penndu** [comment]
hexo的algolia就很不好用！

**Comment by caibingcheng** [comment]
> @penndu
> hexo的algolia就很不好用！

我感觉你的搜索功能很好用呀，我用的这个是先分词再构建的index，所以搜索预览的时候就只能看到一个一个单词，没有完整的句子
